### ISSUE TITLE
Using with ES5 browsers?

### ISSUE THREAD

**Issue by dartess** [pbeshai/use-query-params#15]
Hi! 

Unfortunately, I could not use this hook in older browsers. After assembly, the code contains `conts` that do not work.

I tried to add a package to the `rules` `include` section:
```
{
    test: /\.(js|jsx)$/,
    include: [
        path.resolve(__dirname, 'src'),
        path.resolve(__dirname, 'node_modules', 'use-query-params'),
    ],
    use: [
        'babel-loader',
    ],
},
```

But the result has not changed. What am I doing wrong? There were no similar problems with other npm packages.

**Comment by rshackleton** [comment]
Have the same issue, added rules for these plugins by process of trial and error, however I no longer see any console errors on page load but nothing loads. Not sure if it is this plugin or something else at this stage.

```js
{
  test: /\.js$/,
  use: ['babel-loader'],
  include: /query-string/,
},
{
  test: /\.js$/,
  use: ['babel-loader'],
  include: /serialize-query-params/,
},
{
  test: /\.js$/,
  use: ['babel-loader'],
  include: /split-on-first/,
},
{
  test: /\.js$/,
  use: ['babel-loader'],
  include: /strict-uri-encode/,
},
{
  test: /\.js$/,
  use: ['babel-loader'],
  include: /use-query-params/,
},
```

**Comment by rshackleton** [comment]
Quick update, managed to resolve the issues I was having.

1. Make sure you're using `use-query-params@^0.3.2`. This is precompiled for es5 and works with IE11. This also removes the need for the extra babel rules I posted above.
2. Add the following to your package.json file, `query-string@6` no longer supports IE11.

```
"resolutions": {
  "query-string": "^5.0.0"
}
```

3. Profit!

**Comment by pbeshai** [comment]
Thank you so much for looking into this! Sorry I've been really busy, but I'll see what it entails to just use an older query-string in the lib and hopefully simplify this situation.

**Comment by dartess** [comment]
@rshackleton the decision with `resolutions` is only suitable for `yarn`=(
I will wait for the decision from @pbeshai .
Thank you all for the reaction.

**Comment by dimabory** [comment]
@pbeshai Any updates from your side regarding `query-string` version?

**Comment by dimabory** [comment]
> Thank you so much for looking into this! Sorry I've been really busy, but I'll see what it entails to just use an older query-string in the lib and hopefully simplify this situation.

Looking at the code https://github.com/sindresorhus/query-string/compare/v5.1.1...v6.0.0 I noticed that nothing was changed in terms of the main functionality (the commit name complete explains the reason for released 6.0.0 version). So I believe `serialize-query-params` can be safely downgraded to   `"query-string": "^5.0.0"`

**Comment by rshackleton** [comment]
@dimabory I'm using this library with `query-string@^5.0.0` in production and no issues so it is definitely compatible.

**Comment by pbeshai** [comment]
Ok should be fixed in v0.3.3, thank you everyone!
